roles/kube-init/tasks/main.yml

```diff
@@ -13,6 +13,12 @@
     arg_pod_network: "--pod-network-cidr {{ pod_network_cidr }}/16"
   when: pod_network_type != "weave" and not ipv6_enabled
 
+  # This seems to be required for kube-dns (TODO: need investigation)
```

Comment:
Probably worth opening an issue for this, and linking here, just so we don't leave TODO items laying around without them being tracked.

Reply:
I've add issue ticket.